openmls/src/group/tests/test_external_commit_validation.rs

```diff
@@ -586,37 +590,32 @@ fn test_valsem244(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider
         .payload()
         .clone()
         .sign(backend, &bob_credential_bundle)
-        .expect("Error signing modified payload.");
+        .unwrap();
 
     // Set old confirmation tag
-    signed_plaintext.set_confirmation_tag(
-        original_plaintext
-            .confirmation_tag()
-            .expect("no confirmation tag on original message")
-            .clone(),
-    );
+    signed_plaintext.set_confirmation_tag(original_plaintext.confirmation_tag().unwrap().clone());
 
     let verifiable_plaintext: VerifiableMlsAuthContent =
         VerifiableMlsAuthContent::from_plaintext(signed_plaintext, None);
 
-    // Have alice process the commit resulting from external init.
+    // Have Alice process the commit resulting from external init.
     let message_in = MlsMessageIn::from(verifiable_plaintext);
 
     let err = alice_group
         .process_message(backend, message_in)
-        .expect_err("Could process message despite the external commit including an external init proposal by reference.");
+        .unwrap_err();
```

Comment:
Since you removed the string, can you add some comments instead that explain what we test and what we expect?